docs/conf.py

```diff
@@ -13,13 +13,8 @@
 # serve to show the default.
 
 from importlib.metadata import version as get_version
-import sys
-import os
+from pathlib import Path
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.append(os.path.abspath(".."))
```

Comment:
I believe this is a standard placeholder that Sphinx provides, to make it as easy as possible to write custom extensions in future. Did you remove it because you don't think we'll need it any time soon?

Reply:
I just thought it had come from Iris, where it probably used to be active + of value.
If it is part of the "standard" form, I would expect it to be commented out by default?

Reply:
After conversation offline: we're going with the more slimmed-down approach.